cloud-aws/src/main/resources/definitions/aws-audit-policy.json

```diff
@@ -1,24 +1,38 @@
 {
+  "Version": "2012-10-17",
   "Statement": [
     {
-      "Sid": "SimpleStorageServiceFull",
+      "Effect": "Allow",
       "Action": [
-        "s3:AbortMultipartUpload",
-        "s3:CreateBucket",
-        "s3:DeleteObject",
-        "s3:GetObject",
-        "s3:HeadBucket",
         "s3:ListBucket",
-        "s3:GetBucketLocation",
+        "s3:ListBucketMultipartUploads",
+        "s3:GetBucketLocation"
+      ],
+      "Resource": [
+        "arn:aws:s3:::<your-bucket-name-for-audit-archiving>"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:ListMultipartUploadParts",
+        "s3:GetObject",
+        "s3:GetObjectVersion",
         "s3:PutObject",
-        "s3:DeleteBucket",
-        "s3:PutBucketPublicAccessBlock"
+        "s3:DeleteObject"
+      ],
+      "Resource": [
+        "arn:aws:s3:::<your-bucket-name-for-audit-archiving>/*"
+      ]
+    },
+    {
+      "Action": [
+        "ec2:DescribeRegions"
```

Comment:
I don't have this in my own policy, and audit archiving is working fine. I think this is a relatively harmless permission, though, so if you have reason to believe it is necessary, it is fine to leave it in.

Reply:
this is for validate the credential on our side